apps/core/templates/base.html

```diff
@@ -16,6 +16,11 @@
         <meta name="description" content="" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
 
+        {# canonical url #}
+        {% if page.locale.language_code and not page.locale.language_code == "en-latest" %}
+            <link rel="canonical" href="{{ page.full_url|canonical_url:page.locale.language_code }}" >
```

Comment:
I'm not very familiar with SEO, but it seems Google recommends the use of `<link rel="alternate"` for multilanguage sites: https://developers.google.com/search/docs/advanced/crawling/localized-versions

and they also no longer recommend using `<link rel="canonical"`: https://developers.google.com/search/blog/2011/12/new-markup-for-multilingual-content#annotating-pages-as-substantially-similar-content

This page that used to recommend `<link rel="canonical"` now has a warning that points to the "general guidelines for multilingual and multi-regional content" page, (the first link).

Again, I'm not very familiar with SEO, so I might be wrong!

Reply:
@laymonage @Hitansh-Shah Let me run this one of our seo folks at Torchbox and see what they recommend. I'll report back here with my findings.

Reply:
Canonical isn’t meant for this, unless we wanted all languages other than english to not be indexed (which is silly). `rel="alternate"` with a `hreflang` is definitely the way to go.

Something like:

```jinja
<link rel="alternate" hreflang="en" href="{{ page.full_url|canonical_url:page.locale.language_code }}" >
```

This will only create a link from each translated version to english. Normally you’d want each version to link to all other versions, however for the time being the above should be a good start.